sp800-63b/sec5_authenticators.md

```diff
@@ -46,21 +46,21 @@ Verifiers SHOULD offer guidance to the subscriber, such as a password-strength m
 
 Verifiers SHALL implement a rate-limiting mechanism that effectively limits the number of failed authentication attempts that can be made on the subscriber's account as described in [Section 5.2.2](#throttle).
 
-Verifiers SHOULD NOT impose other composition rules (e.g., mixtures of different character types or blocking two repeated characters) on memorized secrets. Verifiers SHOULD NOT require memorized secrets to be changed arbitrarily (e.g., periodically). However, verifiers SHALL force a change if there is evidence of compromise of the authenticator.
+Verifiers SHOULD NOT impose other composition rules (e.g., mixtures of different character types or repeating characters consecutively) on memorized secrets. Verifiers SHOULD NOT require memorized secrets to be changed arbitrarily (e.g., periodically). However, verifiers SHALL force a change if there is evidence of compromise of the authenticator.
```

Comment:
"repeating characters consecutively" isn't a composition rule; it's what they're trying to avoid. Perhaps "prohibiting consecutively repeated characters".